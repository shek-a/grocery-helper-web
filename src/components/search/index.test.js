import React from 'react';
import { configure, shallow } from 'enzyme';
import Adaptar from 'enzyme-adapter-react-16';
import Search from '.';

configure({adapter: new Adaptar()});

describe('search component', () => {
    let shallowWrapper;
    const getWrapper = (props) => shallow(<Search {...props}/>);

    it('should render search', () => {

        let props = {
            categories: ['FRUIT', 'VEGETABLE']
        };

        shallowWrapper = getWrapper(props);
        expect(shallowWrapper).toMatchSnapshot();
    });
})