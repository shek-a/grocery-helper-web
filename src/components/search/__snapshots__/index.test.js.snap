// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search component should render search 1`] = `ShallowWrapper {}`;
